lib/task/jobs/jobWorkerTask.class.php

```diff
@@ -67,6 +67,39 @@ public function getJobsCompleted()
         return $this->jobsCompleted;
     }
 
+    public static function getMemoryUsageString()
+    {
+        return sprintf(
+            "Worker memory usage (PHP - VmRSS): %.2fMb - %dkb\n",
+            self::getPhpReportedMemoryUsage(),
+            self::getLinuxReportedMemoryUsage()
+        );
+    }
+
+    /* Get OS reported mem usage from /prod/self/status.
+     * Grabs the memory value from the VmRSS entry in the status file.
+     * VmRSS is the resident mem set size in kB.
+     *
+     * Returns the resident memory in kB. If the lookup fails this function will
+     * return zero and the caller should deal with that as an error.
+     */
+    public static function getLinuxReportedMemoryUsage()
```

Comment:
Again, I think it's better to use a standard (non-static) method unless there's a specific reason to make it static.  I won't keep reposting the same comment, but I think this applies to the other static methods in the PR as well.

Reply:
See above.